msb/rawlogger/msb_rawlogger.py

```diff
@@ -32,20 +33,25 @@ def get_data(self) -> tuple[bytes, bytes]:
                 continue
             yield topic, data
 
-    def run(self):
-        for topic, data_raw in self.get_data():
-            if self.config.print_stdout:
-                print(f"{topic.decode('utf-8')} : {data_raw.decode('utf-8')}")
 
-            data = b'{"' + topic + b'":' + data_raw + b"}\n"
-            self.logger.write(data)
+    def run(self,excluded_topics):
```

Comment:
Do not pass `excluded_topics` here, use `self.excluded_topics` instead.

Reply:
removed "excluded_topics" as parameter in function call.